lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+var getDefaultOptions = function () {
```

Comment:
From what I understand, `var` is local when inside a module:

https://nodejs.org/api/globals.html

Do we know if this is the same for functions in a module?

with that in mind though, for sure adding `var functionName = function...` should be local to that module - correct?